File: tests/fixtures/simple.go
package main

import "fmt"

func greet(name string) string {
	message := fmt.Sprintf("Hello, %s!", name)
	fmt.Println(message)
	return message
}

func add(a, b int) int {
	result := a + b
	return result
}

func main() {
	name := "World"
	greeting := greet(name)
	_ = greeting
	sum := add(2, 3)
	fmt.Printf("Sum: %d\n", sum)
}

File: tests/fixtures/simple.py
#!/usr/bin/env python3
"""Simple test program for debugging."""


def greet(name: str) -> str:
    """Return a greeting message."""
    message = f"Hello, {name}!"
    return message


def main() -> None:
    """Main entry point."""
    names = ["Alice", "Bob", "Charlie"]
    for name in names:
        greeting = greet(name)
        print(greeting)


if __name__ == "__main__":
    main()
